policies/cli/policy.yaml

```diff
@@ -1,3 +1,19 @@
+# Copyright The Enterprise Contract Contributors
```

Comment:
Nitpick, perhaps `.ec/cli-policy.yml`? Reasoning that `.ec` might be a directory use as a convention?

Reply:
That doesn't work because go-gather expects it to be a directory when using git gather:

```
Error: error copying directory: /tmp/git-repo-3559490554/.ec/cli-policy.yaml is not a directory
```

And the file has to be named `policy.yaml` as well. We should make this better but I'd rather not do that as part of this PR.

I could rename `policies` to `.ec`?